compat/src/render.js

```diff
@@ -161,6 +161,21 @@ options.vnode = vnode => {
 				value = undefined;
 			}
 
+			if (/^oninput$/i.test(i) && normalizedProps.oninput) {
+				const first = normalizedProps.oninput;
+				const next = value;
+				value = function onInput() {
+					const eventHandlerArguments = arguments;
+					[first, next].forEach(callback => {
+						try {
+							callback(eventHandlerArguments);
+						} catch (e) {
+							Promise.reject(e);
```

Comment:
```suggestion
							throw e;
```

I wonder if this is safer

Reply:
If I throw the error, the next callback won't be executed. I can simulate the event queue throwing the exception inside `setTimeout`

```jsx
try {
	callback(evt);
} catch (e) {
	setTimeout(() => { throw e }, 0)
}
```

Reply:
hmm, maybe something like

```
function onInput() {
  let err;
  yourLoop(cb => { try { cb() } catch (e) { err = e } }
  if err { throw err }
}
```

Reply:
I see, that approach would return the last error, would be likely that both callbacks throw an error, so the first callback would fail silently.

Reply:
In that case, `setTimeout` seems the best solution to queue that message https://developer.mozilla.org/en-US/docs/Web/JavaScript/EventLoop#adding_messages

Reply:
![image](https://user-images.githubusercontent.com/11670923/173227397-3b980b25-3061-497c-8483-64077dd4fe84.png)